app/scss/_base.scss

```diff
@@ -14,7 +14,7 @@
 .right {text-align: right;}
 .center {text-align: center;}
 .vertical-space-sm {height: $sm;}
-.vertical-space-xl {height: $xl;}
+.vertical-space-md {height: $md;}
```

Comment:
I realize these vertical space changes affect more than just the section I worked on, but I'll address any fallout in other sections from this change in future PRs.

Reply:
Instead of deleting the class `.vertical-space-xl`, just leave it in until you've confirmed all instances of it are removed from the html.

Reply:
Derr, yes. Excellent point. ✅